src/NavDropdown.js

```diff
@@ -51,28 +51,27 @@ class NavDropdown extends React.Component {
       disabled,
       active,
       ...props
-    } = this.props;
+    },
+    ref,
+  ) => (
+    <Dropdown ref={ref} {...props} as={NavItem}>
+      <Dropdown.Toggle
+        id={id}
+        eventKey={null}
+        active={active}
+        disabled={disabled}
+        childBsPrefix={bsPrefix}
+        as={NavLink}
+      >
+        {title}
+      </Dropdown.Toggle>
 
-    return (
-      <Dropdown {...props} as={NavItem}>
-        <Dropdown.Toggle
-          id={id}
-          eventKey={null}
-          active={active}
-          disabled={disabled}
-          childBsPrefix={bsPrefix}
-          as={NavLink}
-        >
-          {title}
-        </Dropdown.Toggle>
-
-        <Dropdown.Menu role={menuRole} rootCloseEvent={rootCloseEvent}>
-          {children}
-        </Dropdown.Menu>
-      </Dropdown>
-    );
-  }
-}
+      <Dropdown.Menu role={menuRole} rootCloseEvent={rootCloseEvent}>
+        {children}
+      </Dropdown.Menu>
+    </Dropdown>
+  ),
+);
 
```

Comment:
needs a `displayName` explicitly added here

Reply:
I'd be happy to add it, judging from other components it would be
`NavDropdown.displayName = 'NavDropdown';`

Do you have resources/reasoning for why it's needed? I didn't see anything in the component that mentioned it before, and all I can infer is you want to avoid ForwardedRef being the component that shows in an error?